repos/fdbt-site/tests/pages/api/searchOperators.test.ts

```diff
@@ -394,7 +398,76 @@ describe('searchOperators', () => {
             selectedOperators: mockSelectedOperators,
         };
 
-        searchOperators(req, res);
+        await searchOperators(req, res);
+
+        expect(updateSessionAttributeSpy).toHaveBeenCalledWith(
+            req,
+            MULTIPLE_OPERATOR_ATTRIBUTE,
+            expectedSessionAttributeCall,
+        );
+        expect(res.writeHead).toBeCalledWith(302, {
+            Location: '/saveOperatorGroup',
+        });
+    });
+    it("should redirect with errors when the user clicks the 'Confirm operators and continue' button and Bods has no services", async () => {
```

Comment:
the last part of the test needs to be slightly changed, because it's not BODS we are checking, its our database.

` it("should redirect with errors when the user clicks the 'Confirm operators and continue' button and one of the selected operators has no bods services",`